firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/send/DataTransportCrashlyticsReportSender.java

```diff
@@ -46,28 +49,34 @@ public class DataTransportCrashlyticsReportSender {
       (r) -> TRANSFORM.reportToJson(r).getBytes(Charset.forName("UTF-8"));
 
   private final ReportQueue reportQueue;
-  private final Transformer<CrashlyticsReport, byte[]> transportTransform;
+
+  private static final Destination destination =
+      new CCTDestination(CRASHLYTICS_ENDPOINT, CRASHLYTICS_API_KEY);
+  static final TransportContext transportContext =
+      TransportContext.builder()
+          .setBackendName(destination.getName())
+          .setExtras(destination.getExtras())
+          .setPriority(Priority.HIGHEST)
+          .build();
```

Comment:
Hm, for some reason my previous comment on this disappeared, will try to replicate it again:

`TransportContext` as well as `TransportRuntime.getUploader()` are completely internal to firelog and I'd rather we did have teams call them directly. Instead I would suggest to encapsulate this logic with something like the following:

```java
package com.google.android.datatransport.runtime;

@Discouraged(
  message = "TransportRuntime is not a realtime delivery system, don't use unless you absolutely must.")
public final class ForcedSender {
  private ForcedSender() {}

  @WorkerThread
  public static void sendBlocking(Transport<?> transport) {
    TransportContext ctx = castToTransportImplAndGetContextOutOrThrow(transport);
    TransportRuntime.getInstance().getUploader().send(ctx, 1, () -> {});
  }
```

Reply:
Done